PostHog/PostHogApi.swift

```diff
@@ -35,6 +35,26 @@ class PostHogApi {
         return request
     }
 
+    private func getRemoteConfigRequest() -> URLRequest? {
+        guard let baseUrl: URL = switch config.host.absoluteString {
+        case "https://us.i.posthog.com":
+            URL(string: "https://us-assets.i.posthog.com")
+        case "https://eu.i.posthog.com":
+            URL(string: "https://eu-assets.i.posthog.com")
+        default:
+            config.host
+        } else {
+            return nil
+        }
+
+        let url = baseUrl.appendingPathComponent("/array/\(config.apiKey)/config")
```

Comment:
I think we should fix https://github.com/PostHog/posthog-ios/issues/274 otherwise its one more place that this will fail